core/src/main/java/org/pentaho/platform/web/http/api/resources/JobScheduleRequest.java

```diff
@@ -98,7 +98,7 @@ public class JobScheduleRequest implements Serializable, IJobScheduleRequest {
 
   SimpleJobTrigger simpleJobTrigger;
 
-  List<JobScheduleParam> jobParameters = new ArrayList<>();
+  List<IJobScheduleParam> jobParameters = new ArrayList<>();
```

Comment:
interface `IJobScheduleRequest` has getters/setters for List<IJobScheduleParam> decided to correct it here in this PR.